types/object.go

```diff
@@ -285,3 +285,33 @@ func (o Object) IsNull() bool {
 func (o Object) IsUnknown() bool {
 	return o.Unknown
 }
+
+func (o Object) String() string {
+	if o.Unknown {
+		return attr.UnknownString
+	}
+
+	if o.Null {
+		return attr.NullString
+	}
+
+	// We want the output to be consistent, so we sort the output by key
+	keys := make([]string, 0, len(o.Attrs))
+	for k := range o.Attrs {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	var res strings.Builder
+
+	res.WriteString("(")
```

Comment:
Terraform tends to use curly braces (e.g. `{`) for objects, so maybe best to stick with that to prevent confusion. 👍